### PR TITLE
[bitnami/minio] Fix readiness config

### PR DESCRIPTION
**Pull request by mschae**
**Description of the change**

This fixes the configuration of the helm chart `minio` through helm values.

1. In the template of the standalone deployment the chart mistakingly uses the `livenessProbe` values in the `readinessProbe`.
1. The `ingress.enabled` variable is not documented in the README

**Benefits**

The chart behaves as advertised and documents the `ingress.enabled` variabel.

**Possible drawbacks**

Configuration will change for users that have previously used the wrong template.

**Applicable issues**

None found.

**Additional information**

None

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Comment by alemorcuq** [comment]
Thanks for the PR, @mschae. There are some commits that don't belong here, could you fix it?

**Comment by mschae** [comment]
@alemorcuq Sorry about that, I've only been using git for 10 years. Seems like I yet have to figure out rebasing.

File: bitnami/minio/README.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ The following table lists the configurable parameters of the MinIO chart and the
 | `service.nodePort`                   | Port to bind to for NodePort service type                                                                                                                 | `nil`                                                   |
 | `service.loadBalancerIP`             | Static IP Address to use for LoadBalancer service type                                                                                                    | `nil`                                                   |
 | `service.annotations`                | Kubernetes service annotations                                                                                                                            | `{}`                                                    |
+| `ingress.enabled`                    | Enable/diasable ingress                                                                                                                                   | `false`                                                 |
```
Comment (alemorcuq) [inline bitnami/minio/README.md]: Type: `diasable` -> `disable`

**Comment by alemorcuq** [comment]
Thanks!

**Pull request merged**